examples/e-commerce/src/widgets/Pagination.js

```diff
@@ -3,7 +3,7 @@ import { pagination as paginationWidget } from 'instantsearch.js/es/widgets';
 const pagination = paginationWidget({
   container: '#pagination',
   scrollTo: '.container',
-  totalPages: 4,
+  padding: 2,
```

Comment:
With a `padding` we probably want to display first & last page. Otherwise it's cumbersome to go back on the first page with only previous arrow.

Reply:
That's not part of the design and this will break it.

Reply:
Yes I know that it's not, worth asking Nico though.